lib/imagec/imagec.go

```diff
@@ -138,7 +138,7 @@ var (
 
 const (
 	// DefaultDockerURL holds the URL of Docker registry
-	DefaultDockerURL = "registry.hub.docker.com"
+	DefaultDockerURL = "registry-1.docker.io"
```

Comment:
Really minor, but: given that we already have the docker code vendored, should we consider setting this value equal to the constant from their codebase?

https://github.com/vmware/vic/blob/24af8e8e5536ca0bcf7ee2a9e659822585654b4a/vendor/github.com/docker/docker/registry/config.go#L50-L53

It seems like that makes the relationship between this code and the docker client code a bit more obvious, hopefully reducing the chances that this constant is incorrectly changed in the future.